memegen/templates/image.html

```diff
@@ -1,27 +1,33 @@
 <!DOCTYPE html>
 <html>
-<head>
+  <head>
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:site" content="@jasich">
```

Comment:
You should create a memegen twitter handle.  Depending on how engrained your branding is in the project you could make this a variable.